src/Angor/Client/NetworkConfiguration.cs

```diff
@@ -175,6 +175,21 @@ public string GetNetworkNameFromGenesisBlockHash(string genesisBlockHash)
         };
     }
 
+    public List<SettingsUrl> GetDefaultChatAppUrls()
+    {
+        if (currentNetwork.NetworkType == NetworkType.Mainnet)
```

Comment:
For chat you don't have mainnet and testnet (its like relays)

Reply:
ok , I will remove extra code